gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationRequestMapperTest.java

```diff
@@ -478,6 +478,29 @@ void When_MultipleOrdersAreBasedOnOneAcutePlan_Expect_TheLatestOrderPriorPrescri
         );
     }
 
+    @Test
+    void When_MultipleOrdersAreBasedOnOneAcutePlan_Expect_GeneratedPlanDispenseRequestValidityPeriodIsCopiedFromTheLatestOrder() {
```

Comment:
I'm not sure `FromTheLatestOrder` is the requirement is it?

Reply:
```suggestion
    void When_TwoEhrSupplyPrescribeReferenceOneAcuteEhrSupplyAuthorise_Expect_GeneratedPlanDispenseRequestValidityPeriodIsCopiedFromTheLatestOrder() {
```

Reply:
I'm also wondering if a `@Nested` approach would help, if you've got multiple tests with the same When condition.